test/strand-datepicker.html

```diff
@@ -184,13 +184,18 @@
 		dp.end = end;
 
 		flush(function() {
-			var disableFuture = moment(dp.$$('#start').disableFuture);
-			var disablePast = moment(dp.$$('#end').disablePast);
-			start = moment(start);
-			end = moment(end);
-
-			expect(disableFuture.diff(end, 'days')).to.be.lessThan(1);
-			expect(disablePast.diff(start, 'days')).to.be.lessThan(1);
+			var disableFuture = moment(dp.$$('#start').disableFuture).utc();
+			var disablePast = moment(dp.$$('#end').disablePast).utc();
+			start = moment(start).utc();
+			end = moment(end).utc();
+
+			expect(disableFuture.month()).to.equal(end.month());
+			expect(disableFuture.date()).to.equal(end.date());
+			expect(disableFuture.year()).to.equal(end.year());
+
+			expect(disablePast.month()).to.equal(start.month());
+			expect(disablePast.date()).to.equal(start.date());
+			expect(disablePast.year()).to.equal(start.year());
```

Comment:
Because `moment("11/11/2011 0:00 GMT+0600").utc().date() === 10` (and not `11`), I think the conditions for correctness here may still need some more consideration